lib/src/features/symbol_point_renderer.dart

```diff
@@ -58,8 +61,13 @@ class SymbolPointRenderer extends FeatureRenderer {
       }
 
       context.tileSpaceMapper.drawInPixelSpace(() {
+        context.canvas.translate(offset.dx, offset.dy);
```

Comment:
Symbol rotation should be controlled by [text-rotation-alignment](https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#layout-symbol-text-rotation-alignment)

Reply:
Rather than rotating all text, we should be using the style to decide which labels get rotated.

Reply:
From my understanding all points should be rotated, except lines, for lines you are painting them in separate class so it does not matter.

But yes you are right. Styles should decide if symbol is rotate able.